tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from httpx import AsyncClient
+
+from routes import app
+
+
+@pytest.fixture(scope="module")
+async def client():
+    async with AsyncClient(app=app, base_url="http://localhost:8000") as client:
```

Comment:
should id be localhost or apiUrl value?

Reply:
This should be `localhost`, because the tests will be run on the local machine, and they are expecting to find the local API on the `localhost` address